pkg/network/server.go

```diff
@@ -517,6 +517,29 @@ func (s *Server) handleInvCmd(p Peer, inv *payload.Inventory) error {
 	return nil
 }
 
+// handleMempoolCmd handles getmempool command.
+func (s *Server) handleMempoolCmd(p Peer) error {
+	txs := s.chain.GetMemPool().GetVerifiedTransactions()
+	hs := make([]util.Uint256, 0, payload.MaxHashesCount)
+	for i := range txs {
+		hs = append(hs, txs[i].Hash())
+		if len(hs) < payload.MaxHashesCount && i != len(txs)-1 {
+			continue
+		}
+		inv := payload.NewInventory(payload.TXType, hs)
+		pkt, err := NewMessage(CMDInv, inv).Bytes()
+		if err != nil {
+			return err
+		}
+		err = p.EnqueueHPPacket(pkt)
```

Comment:
Why is it an HP packet?

Reply:
Fixed. In C# it is `mempool` command itself which is HP.